hw7ag/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace HW7_again
+{
+    class Program
+    {
+        static void Main()
+        {
+            
+            Console.WriteLine("Введите значение X:");
+           double x = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("Значение функции равно" + GetValueOfFunction(x));
```

Comment:
Лучше отделить текст от выводимого значения пробелом
```suggestion
            Console.WriteLine("Значение функции равно " + GetValueOfFunction(x));
```